docs/resources/Channel.md

```diff
@@ -281,7 +281,7 @@ Represents a message sent in a channel within Discord.
 ###### Message Types
 
 > warn
-> Type `19` and `20` are only in API v8. In v6, they are still type `0`.  Type `21` is only in API v9.
+> Type `19` and `20` are only in API v8 and higher. In v6, they are still type `0`.  Type `21` is only in API v9 and higher.
```

Comment:
```suggestion
> Type `19` and `20` are only available in API v8 and higher. In v6, they are represented as type `0`.  Additionally, type `21` is only available in API v9 and higher.
```

Reply:
Added in 15abf0aa376c3d893bdf02432e40206d2cbdf946 -- thanks for taking a look!